.github/workflows/ci.yml

```diff
@@ -0,0 +1,48 @@
+name: CI
+
+on:
+  push:
+    branches:
+    - 'ci/**'
+    - 'master'
+    - 'develop'
+  pull_request:
+    branches:
+      - '**'
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
```

Comment:
Do we want to control the node version used with something like https://github.com/actions/setup-node?